### PR TITLE
operator [R] keycloak-operator (12.0.1 12.0.3 13.0.0 13.0.1 14.0.0 15.0.0 15.0.1 15.0.2 15.1.0 15.1.1 16.0.0 16.1.0 17.0.0 17.0.1 18.0.0 18.0.1 18.0.2 19.0.0 19.0.1 19.0.2 19.0.3 20.0.0 20.0.0-alpha.1 20.0.0-alpha.2 20.0.0-alpha.3 20.0.0-alpha.4 20.0.0-alpha.5 20.0.0-alpha.6 20.0.0-alpha.7 20.0.1 20.0.2 20.0.3 21.0.0 21.0.1 21.0.2 21.1.0 21.1.1 21.1.2 22.0.0 22.0.1 22.0.2 22.0.3 22.0.4)

### PR DESCRIPTION
**Pull request by pierreprinetti**
Before this patch, clicking on the documentation link resulted in a 404 Not Found on Github pages.

With this change, the documentation link points to the "Operator" anchor in the Keycloak docs index. Note that the operator docs don't seem to be versioned on the Keycloak website.

This patch is the result of:

```sh
find . -name '*keycloak-operator*.yaml' -exec sed -i 's|\(\s\+url: \)https://www.keycloak.org/docs/latest/server_installation/index.html#_operator\s*$|\1https://www.keycloak.org/guides#operator|' {} \;
```

### CODE REVIEW

**Comment by pierreprinetti** [comment]
cc @vmuzikar

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@pierreprinetti Thanks for the fix! Could you please rather update the [source](https://github.com/keycloak/keycloak/blob/058d00fea849a7c9ffdfa7d35aed4d324579199d/operator/src/main/java/org/keycloak/operator/controllers/KeycloakSharedCsvMetadata.java#L44) that is used for generating the CSV? We won't be fixing this retrospectively.

**Comment by pierreprinetti** [comment]
@vmuzikar Here it is: https://github.com/keycloak/keycloak/pull/23777

As for the backports, I would argue that users of the previous versions may appreciate a working docs link :shrug: 
However, that's your call of course.

Have a nice day!

**Comment by pierreprinetti** [comment]
I noticed that many Italians have the same exact representation of that specific frame of Conway's Game of Life's spaceship as their avatar here on Github. Would you know why, @mporrato?

**Comment by mporrato** [comment]
> I noticed that many Italians have the same exact representation of that specific frame of Conway's Game of Life's glider spaceship as their avatar here on Github. Would you know why, @mporrato?

@pierreprinetti LOL, I haven't noticed the pattern you mention with fellow Italian github users, tbh, but being the "de facto" hacker symbol it's very common in general in this kind of communities. Perhaps we are all very unimaginative :shrug:

**Comment by mporrato** [comment]
@vmuzikar do you want this merged?

**Comment by vmuzikar** [comment]
@mporrato Please don't merge. We'll address this via https://github.com/keycloak/keycloak/pull/23777

**Comment by mporrato** [comment]
@vmuzikar thanks. Closing.

**Pull request closed without merging**
